testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/Secrets.java

```diff
@@ -0,0 +1,130 @@
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+package com.microsoft.identity.internal.testutils.labutils;
+
+import com.microsoft.identity.common.exception.ClientException;
+import com.microsoft.identity.common.internal.authorities.AccountsInOneOrganization;
+import com.microsoft.identity.common.internal.authorities.AzureActiveDirectoryAuthority;
+import com.microsoft.identity.common.internal.providers.keys.CertificateCredential;
+import com.microsoft.identity.common.internal.providers.keys.ClientCertificateMetadata;
+import com.microsoft.identity.common.internal.providers.keys.KeyStoreConfiguration;
+import com.microsoft.identity.common.internal.providers.microsoft.MicrosoftClientAssertion;
+import com.microsoft.identity.common.internal.providers.microsoft.microsoftsts.MicrosoftStsTokenRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenResult;
+import com.microsoft.identity.internal.test.keyvault.Configuration;
+import com.microsoft.identity.internal.test.keyvault.api.SecretsApi;
+import com.microsoft.identity.internal.test.keyvault.model.SecretBundle;
+
+import java.io.IOException;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.UnrecoverableKeyException;
+import java.security.cert.CertificateException;
+
+/**
+ * This class contains helper methods to obtain passwords from KeyVault using the {@link SecretsApi}
+ */
+public class Secrets {
+
+    private static String mAccessToken = null;
+    public static String API_VERSION = "2016-10-01";
+    private final static String CLIENT_ID = "4bc6e96f-bd23-408f-8ecb-a7a7145463f9";
```

Comment:
Just checking that you are intentionally including this client id in this PR.

Confirm?

Reply:
This is actually old code, I just moved it from the adal repo to the common repo.

See adal repo: https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1456 
See common repo: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/609

This CLIENT_ID is used to make the request to the KeyVault and I believe it is necessary.